orchestrator/graphql/types.py

```diff
@@ -116,3 +116,14 @@ class GraphqlFilter:
     IPv6Interface: IPv6InterfaceType,
     int: IntType,
 }
+
+
+@strawberry.type(description="Generic class to capture errors")
+class MutationError:
+    message: str = strawberry.field(description="Error message")
+    details: str | None = strawberry.field(description="Details of error cause", default=None)
+
+
+@strawberry.type(description="Error class if a resource couldn't be found (404)")
+class NotFound(MutationError):  # noqa: N818
+    pass
```

Comment:
I suggest to suffix all error types with Error just to make it easily distinguishable. It's a relatively common pattern as well

```suggestion
class NotFoundError(MutationError):  # noqa: N818
    pass
```